application/testing/CMakeLists.txt

```diff
@@ -94,8 +94,13 @@ function(f3d_test)
     set_tests_properties(f3d::${F3D_TEST_NAME} PROPERTIES FIXTURES_REQUIRED f3d::${F3D_TEST_DEPENDS}_FIXTURE)
   endif()
 
+  list(APPEND f3d_test_env_vars "CTEST_F3D_PROGRESS_BAR_TESTING=1")
+  if (F3D_TEST_NO_FILE_RENDER_TESTING)
+    list(APPEND f3d_test_env_vars "CTEST_F3D_NO_FILE_RENDER_TESTING=1")
```

Comment:
about the naming, the `TESTING` suffix sound a bit redundant with the `CTEST` prefix.

Reply:
Here are my suggestions:
- `CTEST_F3D_FORCE_PROGRESS_BAR`
- `CTEST_F3D_NO_FILE_RENDER`

Reply:
good suggestion